### PR TITLE
Readability cleanup

### PR DESCRIPTION
**Pull request by weableandbob**
Makes a few readability improvements to several .cs files and fixes one typo. No functional changes.

### CODE REVIEW

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/ListViewColumnSorter.cs [hunk 1]
```diff
@@ -41,33 +41,40 @@ public ListViewColumnSorter()
...
         /// <param name="x">First object to be compared</param>
         /// <param name="y">Second object to be compared</param>
-        /// <returns>The result of the comparison. "0" if equal, negative if 'x' is less than 'y' and positive if 'x' is greater than 'y'</returns>
+        /// <returns>The result of the comparison. "0" if equal, negative if 'x' is less than 'y' and positive if 'x'
+        /// is greater than 'y'</returns>
         public int Compare(object x, object y)
         {
             int compareResult;
             ListViewItem listviewX, listviewY;
+            string xColumnText, yColumnText;
```
Comment (ThiconZ) [inline Firefall DISASM Name Manager/Firefall DISASM Name Manager/ListViewColumnSorter.cs]: These should be renamed to columnTextX and columnTextY to follow the naming convention already set for this specific file/class.

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs [hunk 2]
```diff
@@ -155,15 +168,21 @@ public int AddNamesToListView(List<NameClass> NamesObject, bool UpdateFullDuplic
                 DeduplicationForm deduplicationForm = new DeduplicationForm(DuplicateObjects, SourceObjects);
                 if (deduplicationForm.ShowDialog(this) == DialogResult.OK)
                 {
-                    // TODO: Currently this is adding another copy of the "deduplicated" name to the list of what to add
-                    // This instead needs to be updating the exist entry
+                    // TODO: Currently this is adding another copy of the "deduplicated" name to the list of what to
+                    // add. This instead needs to be updating the exist entry.
```
Comment (ThiconZ) [inline Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs]: Move "add." back up to the end of the previous line.
Change "updating the exist entry" to say "updating the existing entry" to fix the typo originally made.

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs [hunk 3]
```diff
@@ -82,8 +86,16 @@ public int AddNamesToListView(List<NameClass> NamesObject, bool UpdateFullDuplic
 
             List<NameClass> UpdateObjects = new List<NameClass>();
 
-            Dictionary<string, List<NameClass>> DuplicateObjects = new Dictionary<string, List<NameClass>>() { { "Address", new List<NameClass>() }, { "Name", new List<NameClass>() } };
-            Dictionary<string, List<NameClass>> SourceObjects = new Dictionary<string, List<NameClass>>() { { "Address", new List<NameClass>() }, { "Name", new List<NameClass>() } };
+            Dictionary<string, List<NameClass>> DuplicateObjects = new Dictionary<string, List<NameClass>>()
```
Comment (ThiconZ) [inline Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs:85]: Keep this change.

**Comment by ThiconZ** [comment]
Thanks for the pull request. Looking over these changes, most if not all the 120 character line style changes can be reverted. On a 1920x1080 at 100% DPI running Visual Studio 2019 General Environment, you can fit 200 characters on a line - assuming you leave things such as the Solution Explorer docked on the screen. Undocking that and other elements will fit at least 250 characters on a line. Other IDEs can fit more on the screen due to having less clutter as well. Overall, we don't care much for characters per line unless it's something truly egregious at which point that line should be reviewed to see why it is so long and what can be done to properly shorten it without causing negative impact to performance or other areas. There’s no need to be wasting vertical screen space.

I’ve left a couple comments on the change for specific additional changes to make/changes to keep. I am currently on the fence about the `MessageBox` related changes. At the very least, the first parameter should be on the same line as the function call (for everything). For now, I would say to revert the `MessageBox` changes in this, and make a new commit that only changes them so it’s easier to decide if it’s worth going through with changing them or not. The rest of the changes appear to just be breaking around the 120 character mark, so they can probably be reverted due to the previously mentioned reasoning.

**Comment by weableandbob** [comment]
Alright, I'll get on reverting most of the changes. I would personally argue that the limit should be significantly under whatever the max is that can fit on a single screen to make it better for having multiple files open side-by-side, but that's a conversation for another time, if ever.

Can some of the basic style guidelines be formalized in the documentation repo (variable naming, line length, etc.)? It would be helpful for new contributors like me. Additionally, in my experience, clear and consistently followed styling helps a lot with long-term maintainability, even more so if you take advantage of readily available automation.

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/ListViewColumnSorter.cs (hunk 1 above)
Comment (weableandbob) [reply Firefall DISASM Name Manager/Firefall DISASM Name Manager/ListViewColumnSorter.cs]: Done.

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs (hunk 2 above)
Comment (weableandbob) [reply Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs]: Done.

File: Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs (hunk 3 above)
Comment (weableandbob) [reply Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs:85]: Ack.

**Comment by weableandbob** [comment]
Line length changes have been reverted.

**Comment by ThiconZ** [comment]
Thanks for the PR, the changes have been merged now.

As for style guidelines, that is something we can look at creating and adding to the Wiki at some point. We don't currently have an actual set of guidelines that we follow across the entire team. For now I would suggest to just read through each code base you plan to contribute to in order to familiarize yourself with what styles appear to be getting used. While we don't have an official team-wide style right now, pretty much all of our projects follow very similar styles. Additionally, we do tend to follow the Allman Indentation Style. Our current concerns are more geared towards first getting out working tools that have a clear and maintainable codebase and when needed, can go back and standardize them.

**Pull request merged**